tsconfig.json

```diff
@@ -1,11 +1,11 @@
 {
 	"compilerOptions": {
+		"esModuleInterop": true,
 		"strict": false,
 		"types": [
-			"chrome"
+			"node"
```

Comment:
why "node" here and "chrome" in the inner one? also where are we using node types

Reply:
we use node to execute webpack!

Reply:
oop okey